### DISCUSSION TITLE
Windows 11 22621.3007  and explorer patcher build 22621.3296.64.1 "latest version"

### DISCUSSION THREAD

**Discussion by 0s9df8c0sdfd0** [valinet/ExplorerPatcher#3127] [Q&A]
I wanted to install Explorer Patcher again and I found out downloading the latest version works. But I want to know if Windows 11 build 22621.3007 Will work with no glitches (like black screen on startup black desktop destroyed taskbar destroyed everything) With Explorer Patcher build 22621.3296.64.1. I want to know this because I don't want my PC bricked due to explorer patcher.

**Comment by pyrates999** [comment]
Update your windows version first with the windows updates released on April 9.

**Comment by 0s9df8c0sdfd0** [comment]
Also for some reason. I can't even update my PC to a February update. And
update to the April version.

**Comment by pyrates999** [comment]
You may need to repair your windows install.  Please run the following in an elevated command prompt or elevated powershell:

```
chkdsk C:
```

If it tells you there are errors, then run this:

```
chkdsk C: /F
```

Reboot to run it on boot.  Keep running that command until it reports no errors.  Then run this to repair windows:

```
sfc /scannow
dism /online /cleanup-image /scanhealth
dism /online /cleanup-image /restorehealth
dism /online /cleanup-image /checkhealth
dism /online /cleanup-image /analyzecomponentstore
dism /online /cleanup-image /startcomponentcleanup
```

Then reboot your computer.
